tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapPropertiesTest.java

```diff
@@ -126,7 +127,9 @@ private void verifyTiledMap (TiledMap tiledMap) {
 	}
 
 	private <T> void verifyProperty (String propName, T expected, T actual) {
-		if (!Objects.equals(expected, actual)) {
+		if (expected instanceof Float && !MathUtils.isEqual((Float) expected, (Float) actual, 0.1f)) {
```

Comment:
I tried to run GWT but have no experience with it and gave up.

Are there some easy instructions to get it run?

I saw this gwtRun Gradle task or something which opened a window but clicking in it always said I need to compile GWT first. The gwtCompile Gradle task did not help.

Please let me know how to run this test there. Makes it easier to understand and fix.

Reply:
Usually the `superDev` task (in the test module for GWT) is enough to see if it compiles and runs as expected. It runs until you end the task, but after probably less than a minute of initially compiling it will say it's ready and make a server available. The link to the server page is just a bit above the bottom, it's some localhost or 127.0.0.1 link with a specific port... I can't remember off the top of my head. There is another server link that takes you to a configuration page with bookmarklets you can use to recompile changes from inside the browser, when you're on a superdev app's page (that link probably isn't needed here). Once you open the non-configuration link, it should load normally and show the list of tests; if not, there's some runtime error. I can find this info more specifically/with more accuracy if needed.

Reply:
tommyettinger is correct and as long as you don't run into any weird errors.
It should be fairly simple once you get it going the first time.
Example. Manually running the task from terminal like so:
./gradlew tests:gdx-tests-gwt:superDev
Example gif of output (sped up):
![superDev](https://github.com/user-attachments/assets/aa77933b-0738-4f69-bbfc-814eeae7f58e)
You will end up with 2 links in the log. In my case:
"The code server is ready at http://127.0.0.1:9876" which should bring you here:
![image](https://github.com/user-attachments/assets/b6a790f7-1476-4245-82fc-0f3004a958ca)
and "2024-12-03 08:51:07.858:INFO:oag.JettyConfigurerImpl:main: Jetty 9.4.24.v20191120 started and listening on port 8080
2024-12-03 08:51:07.875:INFO:oag.JettyConfigurerImpl:main:  runs at:
2024-12-03 08:51:07.875:INFO:oag.JettyConfigurerImpl:main:   http://localhost:8080/
"
which should bring you here to the test page which looks like this:
![image](https://github.com/user-attachments/assets/72465c2e-c6a7-402c-9338-0c56fe4b7b10)
Note you might need to recompile, or compile for the first if you hadn't done so. Using that little refresh icon.

And then just hope it loads and works. Good luck!

Reply:
thank you both! got it to work with the superdev task and then simply connecting to localhost:8080 and then I saw all the tests.